case-studies/README.md

```diff
@@ -0,0 +1,24 @@
+# Case Studies
+
+## Objective
+
+SIG-Security work ought to be informed by data and feedback on what works and what doesn't.
+This will help create a feedback loop between the theoretical/hypothetical and practical/tactical.
+This falls into 2 categories: process examples (e.g. how well did an assessment go?) and security practices examples (irrespective of the assessment, how are tools working out in the wild?)
+
+## Vision
+* Document example case studies, both uses of where things went right, and where things went wrong
+* Support knowledge-sharing that reduces security risks, providing data about vendors and open source projects
+* Define a process for "admission control" for what a case study should look like to add value (not too detailed, not too hand-wavy)
+
+# Suggested Case Studies
+
+## Process Case Studies
+* OPA Assessment
+* In-Toto Assessment
```

Comment:
"in-toto" is all lower-case -- can you fix here and below?

see comment above -- maybe leave out this section, and we can add back in if/when the security assessment team creates any of these.  I don't want to imply that they need to add a step in their process to prepare more documentation.

Reply:
No the idea isn't that the assessment team creates the case study; someone who did not do the assessment, or part of the project team, should write these up and give unbiased feedback on what worked and what didn't.

The case study is separate and apart from the assessment itself. it is meta.

ie the assessments try to result in awesomeness, and the case studies are assessments of the assessments to ensure they are resulting in awesomeness!